components/slider/index.css

```diff
@@ -455,17 +482,269 @@ governing permissions and limitations under the License.
 /* backwards compatibility: these are not required, but they make the handle go to the edegs and align right */
 .spectrum-Slider-handleContainer,
 .spectrum-Slider-trackContainer {
-  width: calc(100% + var(--spectrum-slider-handle-width));
+  width: calc(100% + var(--spectrum-slider-handle-size));
 
   position: absolute;
-  top: calc(var(--spectrum-slider-track-height) / 2 - 1px);
+  top: calc(var(--mod-slider-control-height, var(--spectrum-slider-control-height)) / 2 - 1px);
 
-  margin-left: calc(var(--spectrum-slider-handle-width) / 2 * -1);
+  margin-left: calc(var(--spectrum-slider-handle-size) / 2 * -1);
```

Comment:
```suggestion
  margin-inline-start: calc(var(--spectrum-slider-handle-size) / 2 * -1);
```

Reply:
☝🏻 as this is, when switching to RTL, the track is incorrectly positioned. I think switching to the logical property will resolve that.

<img width="314" alt="Screen Shot 2023-01-18 at 4 28 52 PM" src="https://user-images.githubusercontent.com/360251/213299063-071a1184-63c9-48e0-87e2-5b55106d32c7.png">

Reply:
Spot on @pfulton .
I am getting to the point where paddings and margins are a rare exception to the inline-start for RLT.   
<img width="363" alt="Screen Shot 2023-01-23 at 13 34 33" src="https://user-images.githubusercontent.com/52184321/214144192-653e3220-f1e7-457c-bc86-1974fbc7575a.png">